modules/alphamat/src/intraU.cpp

```diff
@@ -0,0 +1,136 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+#include "intraU.hpp"
+
+namespace cv{
+  namespace alphamat{
+
+std::vector<int> orig_ind;
```

Comment:
Don't use global variables.

Create class with fields for storing algorithm's state.

Reply:
Removed the global parameter..passed it using references between functions.